src/snyk/snykCode/constants/dcignore.ts

```diff
@@ -0,0 +1,53 @@
+export declare const MAX_PAYLOAD: number;
+export declare const MAX_FILE_SIZE: number;
+export declare const HASH_ALGORITHM = 'sha256';
+export declare const ENCODE_TYPE = 'hex';
+export declare const GIT_FILENAME = '.git';
+export declare const GITIGNORE_FILENAME = '.gitignore';
+export declare const DCIGNORE_FILENAME = '.dcignore';
+export declare const DOTSNYK_FILENAME = '.snyk';
+export declare const EXCLUDED_NAMES: string[];
+export declare const CACHE_KEY = '.dccache';
+export declare const MAX_UPLOAD_ATTEMPTS = 10;
+export declare const UPLOAD_CONCURRENCY = 2;
+export declare const POLLING_INTERVAL = 500;
+export declare const MAX_RETRY_ATTEMPTS = 10;
+export declare const REQUEST_RETRY_DELAY: number;
+export declare const ORG_ID_REGEXP: RegExp;
+export declare const IGNORES_DEFAULT: string[];
+export declare const IGNORE_FILES_NAMES: string[];
+export declare const DCIGNORE_DRAFTS: {
```

Comment:
I think you may have copied these over and formatted them incorrectly from https://github.com/snyk/code-client/blob/master/src/constants.ts. I was following https://docs.snyk.io/snyk-scm-ide-and-ci-cd-integrations/use-snyk-in-your-ide/visual-studio-code-extension/create-a-.dcignore-file to try and use this command that gets run with the constants and I'm seeing an error
<img width="1713" alt="Screenshot 2024-06-10 at 12 46 31" src="https://github.com/snyk/vscode-extension/assets/81559517/7652c724-25b9-4d67-abde-30c45d56f6e9">

This file here defined the types of these constants but does not assign a value to them. When I copy that file directly over to this file it works so it should be an easy fix